src/components/Footer.tsx

```diff
@@ -1,21 +1,29 @@
+import { createSharedPathnamesNavigation } from 'next-intl/navigation';
+import { locales } from '@/locales';
 import { useTranslations } from 'next-intl';
 
+const { Link } = createSharedPathnamesNavigation({
+  locales: [...locales.values()],
+});
+
 export default function Footer({ className }: { className?: string }) {
   const t = useTranslations('Footer');
   return (
-    <footer className={`footer bg-black py-2 d-flex flex-wrap ${className ?? ''}`}>
+    <footer className={`footer bg-black py-3 d-flex flex-wrap ${className ?? ''}`}>
       <div className="container-sm d-flex flex-wrap justify-content-around">
         <a
```

Comment:
Please always use the `Link` component provided by Next.js over `<a>` tags. For examples, check out the `Navigation` component.